src/grpc-network-client.ts

```diff
@@ -68,6 +70,23 @@ class GRPCNetworkClient implements Networking.NetworkClient {
       );
     });
   }
+
+  public async getLatestValidatedLedgerSequence(
+    getLatestValidatedLedgerSequenceRequest: GetLatestValidatedLedgerSequenceRequest
+  ): Promise<LedgerSequence> {
+    return new Promise((resolve, reject): void => {
+      this.grpcClient.getLatestValidatedLedgerSequence(
+        getLatestValidatedLedgerSequenceRequest,
+        (error, response): void => {
+          if (error != null || response == null) {
```

Comment:
`!= null` and `== null` checks if the left side value is `null or undefined` if you want to strictly check `null` then use `===`